Kbs.Business/User/UserEntity.cs

```diff
@@ -5,7 +5,7 @@ public class UserEntity
     public int UserId { get; set; }
     public string Email { get; set; }
     public string Password { get; set; }
-    public string Name { get; set; }
+    public string Name { get; set; } = string.Empty;
```

Comment:
This can be removed when other fix is used (in Kbs.Wpf/User/Registration/RegistrationWindow.xaml.cs)